src/containers/forgotPasswordReset/index.tsx

```diff
@@ -15,20 +15,16 @@ const ForgotPasswordReset: React.FC = () => {
   const { key } = useParams();
 
   const onFinish = (values: NewPasswords) => {
-    if (values.password.length < 8) {
-      alert('New password is too weak. Must be at least 8 characters long.');
-    } else if (values.password !== values.confirmPassword) {
-      alert("Passwords don't match");
-    } else {
-      authClient
-        .forgotPasswordReset({ secretKey: key, newPassword: values.password })
-        .then(() => {
-          alert('Successfully reset password!');
-        })
-        .catch((err) => {
-          alert('Was not able to reset password.');
-        });
-    }
+    authClient
+      .forgotPasswordReset({ secretKey: key, newPassword: values.password })
+      .then(() => {
+        window.location.href = '/';
```

Comment:
Instead of this, include `const history = useHistory()` above, and call `history.push(Routes.HOME)`